mindee/endpoints.py

```diff
@@ -1,33 +1,36 @@
 import os
-from typing import Dict, Optional
+from typing import Dict, Optional, Union
 
 import requests
 
 from mindee.input.sources import InputSource
 from mindee.logger import logger
 from mindee.versions import __version__, get_platform, python_version
 
-MINDEE_BASE_URL = "https://api.mindee.net/v1"
-MINDEE_BASE_URL_NAME = "MINDEE_BASE_URL"
-MINDEE_API_KEY_NAME = "MINDEE_API_KEY"
+API_KEY_ENVVAR = "MINDEE_API_KEY"
```

Comment:
I do not think that it is a good idea to name the variable with a technical information.
Moreover, the letter case let think us that it is a const isn't it ? But it is filled from the env variables.

Reply:
These are indeed constants, not variables. I've renamed them to be a bit more explicit as to their function, which is simply to store the name of the environment variable we are looking for.